<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettings</class>
 <widget class="QMainWindow" name="CameraSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="camera_id_lbl">
      <property name="text">
       <string>Camera:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="camera_combo"/>
    </item>
    <item>
     <widget class="QLabel" name="width_lbl">
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="width_sp">
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>640</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="height_lbl">
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="height_sp">
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>480</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="vertical_flip_cb">
      <property name="text">
       <string>Vertical Flip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="horizontal_flip_cb">
      <property name="text">
       <string>Horizontal Flip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="rotate_90_cb">
      <property name="text">
       <string>Rotate 90 Degree </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottom_buttons">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="apply_and_save_btn">
        <property name="text">
         <string>Apply and Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
